### DISCUSSION TITLE
Continious restarting issues with ROG Strix G17

### DISCUSSION THREAD

**Discussion by Afgusto** [seerge/g-helper#2594] [Q&A] [answered]
Hello,

I have searched far and wide across the internet and did not find any suitable answers - if this is not the place, feel free to delete the thread.

**Context:** I have purchased my laptop last year in August and ever since turning it on it keeps hard restarting randomlly (I did not ever consider RMA since I would get it in even worse condition, so not an option for me from the minute I made the purchase). Sometimes it can go a month without doing that, sometimes it chain restarts ten times in a row. Restarts also completely wipes any GHelper settings I had set. Also, any advanced settings I set seem to not have any impact on the system, for example heat control. I can set it to 80C and it will still go to 91C+ even with the setting (maybe I am stupid here, so apologies if anything). I have a feeling I somehow maybe do not grant admin access to the device, idk. 

**Specs:**

CPU: AMD Ryzen 9 7945HX
GPU: 4060
OS: W11

**Troubleshooting I have done so far:**

- I have deleted AC and ASUS bloatware, but I do have Adrenaline installed.
- I have deleted all battery plans and left it on high performance (Ultimate does not appear for some reason no matter what)
- In general I have deleted all the bloatware from Windows
- I have played with multiple GHelper settings, only keeping fans on Turbo seem to provide some kind of results, but the laptop becomes so loud it is not really an option for me. And once again, after each restart any GHelper settings gets wiped.
**- Another interesting thing is any attempt apply Advanced power settings instantly crashes GHelper**
- I did not do a clean install of Windows, maybe I should do it?

Any help would be greatly appreciate, I am very desperate at this point - if there is any logs I can get or provide, please let me know.

**Comment by seerge** [comment]
@Afgusto hello, since you have problems from the beginning no matter what you use AC or G-Helper, it looks more like a faulty hardware. 

You can try to reinstall windows, but i doubt something will change :) I would check if it's possible to repair it.

I can't help you anyhow unfortunately.

**Reply by Afgusto** [reply]
With the restarts - that's fine, maybe I will change thermal paste, do clean install of windows etc.

However, one thing still does not make sense with GHelper - why when trying to apply advanced settings it just simply crashes? I doubt that's faulty hardware issue, maybe you can help me with that? :) Much appreciated!

**Reply by seerge** [reply, marked as answer]
@Afgusto advanced settings is undervolting. If your hardware is already having issues or unstable, it can start crashing even from minimal undervolting. So just don't apply anything there.

**Reply by Afgusto** [reply]
Update - updated BIOS to v329, GHelper no longer crashes with undervolting, will continue monitoring. Next step gonna be reinstalling clean windows and then if that does not help, open it up and change thermal paste or something..

Anyway, to for the help :)

**Comment by Kikimahe** [comment]
Hi,

If I were you, I would save my files and do a clean install of Windows. This often resolves many issues.
And This will allow you to observe how your laptop works with a clean installation of Windows.

If your laptop restarts again, try removing 1 RAM stick (one then the other) to test if it is not a RAM problem.
This happened to me on an old Asus Laptop.

Good luck.

**Reply by Afgusto** [reply]
Thank you for the advice!

**Comment by Uchiha-Jmon** [comment]
i read this on a forum and it seems to pretty much work.

https://rog-forum.asus.com/t5/gaming-notebooks/solved-random-restarts-rog-strix-scar-17-g733pyv-and-other/td-p/1062189

![Screenshot_20250420-041312](https://github.com/user-attachments/assets/0e1ce070-5766-458b-a190-4eed27bca809)
